src/package.xml

```diff
@@ -1,126 +1,96 @@
+      <dataSheet name="outputTabularChange" type="Output" />
+      <dataSheet name="outputSpatialOverall" type="Output" />
+      <dataSheet name="differenceScenarios" type="Input" />
+      <dataSheet name="outputSpatialCategory" type="Output" />
+  </transformer>
+
+  <!--Layouts-->
+
+  <layout type="Scenario" insertAfter="omniscape_Advanced">
+    <group name="Add0on" displayName="Add-on">
+      <group name="Impact0Assessment" displayName="Impact Assessment">
+        <item name="differenceScenarios" showIf="inPipeline" />
+        <group name="Results" displayName="Results">
+          <group name="Spatial" displayName="Spatial">
+            <item name="outputSpatialOverall" showIf="inPipeline" />
+            <item name="outputSpatialCategory" showIf="inPipeline" />
+          </group>
+          <group name="Tabular" displayName="Tabular">
+            <item name="outputTabularDifferences" showIf="inPipeline" />
+            <item name="outputTabularChange" showIf="inPipeline" />
+            <item name="outputTabularJaccard" showIf="inPipeline" />
           </group>
         </group>
       </group>
-    </layout>
-    <layout name="corestimeforms_Charts">
-      <group name="ImpactAssessment" displayName="Impact Assessment">
-        <item name="outputTabularDifferences_amountAreaDifference" displayName="Area difference" dataSheet="outputTabularDifferences" column="amountAreaDifference" filter="movementTypesID" prefixFolderName="False" />
-        <item name="outputTabularDifferences_percentCoverDifference" displayName="Proportion difference" dataSheet="outputTabularDifferences" column="percentCoverDifference" filter="movementTypesID" prefixFolderName="False" />
-        <item name="outputTabularJaccard_dissimilarity" displayName="Jaccard dissimilarity" dataSheet="outputTabularJaccard" column="dissimilarity" filter="movementTypesID" prefixFolderName="False" />
-      </group>
-    </layout>
-    <layout name="corestimeforms_Maps">
-      <group name="ImpactAssessment" displayName="Impact Assessment">
-        <item name="outputSpatialOverall_overallCurrentDifferenceRaster" displayName="Normalized current difference" dataSheet="outputSpatialOverall" column="overallCurrentDifferenceRaster" />
-        <item name="outputSpatialOverall_overallDifferenceRaster" displayName="Cross-category difference" dataSheet="outputSpatialOverall" column="overallDifferenceRaster" />
-        <item name="outputSpatialCategory_perCategoryRaster" displayName="Per category difference" dataSheet="outputSpatialCategory" column="perCategoryRaster" filter="movementTypesID" />
-      </group>
-    </layout>
-  </layouts>
+    </group>
+  </layout>
+
+  <layout type="Chart">
+    <group name="ImpactAssessment" displayName="Impact Assessment">
+      <item name="outputTabularDifferences_amountAreaDifference" displayName="Area difference" dataSheet="outputTabularDifferences" column="amountAreaDifference" filter="movementTypesID" prefixFolderName="False" />
```

Comment:
I don't know if these chart names will work anymore with the "_" - test and fix accordingly